projects/scannet_offline_eval/demo/proto_robot/mock_spot.py

```diff
@@ -0,0 +1,168 @@
+import io
+import logging
+import os
+import pickle
+import random
+import sys
+
+import grpc
+import robodata_pb2
+import robodata_pb2_grpc
+import torch
+from PIL import Image
+from torchvision import transforms
+
+sys.path.append("../../../")
+import home_robot.mapping.voxel.voxel as v
+
+spot_filename = "/private/home/ssax/home-robot/src/home_robot/home_robot/datasets/robot/data/spot_/09_18_fre_spot/spot_output.pkl"
+data = pickle.load(open(spot_filename, "rb"))
+
+img_filename = "proto_robot/dog.webp"
+test_img = Image.open(img_filename)
+
+convert_tensor = transforms.ToTensor()
+center_crop = transforms.CenterCrop((224, 224))
+test_img_tensor = center_crop(convert_tensor(test_img))
```

Comment:
Remove this or add it to a protected `if __name__ == `__main__`?

Reply:
moved to within the methods that are using each (data, test_img)